.vscode/content.code-snippets

```diff
@@ -0,0 +1,47 @@
+{
+    "Chooser": {
+        "prefix": "chooser"
+        "description": "Insert a chooser block for Pulumi code snippets",
+        "body": [
+"{{< chooser language \"typescript,python,go,csharp,java,yaml\" >}}",
+"",
+"{{% choosable language typescript %}}",
+"```ts",
+"${1}",
+"```",
+"{{% /choosable %}}",
+"",
+"{{% choosable language python %}}",
+"```python",
+"${2}",
+"```",
+"{{% /choosable %}}",
+"",
+"{{% choosable language go %}}",
+"```go",
+"${3}",
+"```",
+"{{% /choosable %}}",
+"",
+"{{% choosable language csharp %}}",
+"```csharp",
+"${4}",
+"```",
+"{{% /choosable %}}",
+"",
+"{{% choosable language java %}}",
+"```java",
+"${5}",
+"```",
+"{{% /choosable %}}",
+"",
+"{{% choosable language yaml %}}",
+"```yaml",
+"${6}",
+"```",
+"{{% /choosable %}}",
+"",
+"{{< /chooser >}}"
+        ]
+    }
+}
```

Comment:
We're trying to include JavaScript most everywhere now, so it'd be awesome if we could include that here as well as a nudge to do that. The trailing slash/self-closing mode is the preferred one for the chooser, as it causes it to act "globally" (i.e., to persist the selected language), and the empty lines around the code fences, while not always required, help keep the Markdown parser from going off the rails:

```suggestion
"{{< chooser language \"javascript,typescript,python,go,csharp,java,yaml\" />}}",
"",
"{{% choosable language javascript %}}",
"",
"```javascript",
"${1}",
"```",
"",
"{{% /choosable %}}",
"",
"{{% choosable language typescript %}}",
"",
"```typescript",
"${1}",
"```",
"",
"{{% /choosable %}}",
"",
"{{% choosable language python %}}",
"",
"```python",
"${2}",
"```",
"",
"{{% /choosable %}}",
"",
"{{% choosable language go %}}",
"",
"```go",
"${3}",
"```",
"",
"{{% /choosable %}}",
"",
"{{% choosable language csharp %}}",
"",
"```csharp",
"${4}",
"```",
"",
"{{% /choosable %}}",
"",
"{{% choosable language java %}}",
"",
"```java",
"${5}",
"```",
"",
"{{% /choosable %}}",
"",
"{{% choosable language yaml %}}",
"",
"```yaml",
"${6}",
"```",
"",
"{{% /choosable %}}",
"",
"{{< /chooser >}}"
        ]
    }
}
```

The `typescript` and `javascript` are just for consistency.